niwidgets/ciftitools.py

```diff
@@ -0,0 +1,268 @@
+from __future__ import print_function
+import os, sys, time
+import numpy as np
+import nibabel as nb
+
+#class CiftiTools:
+#    '''
+#    Tools to manipulate cifti files and prepare them for display.
+#    '''
+#
+#    def __init__(self, filename):
+#        if not os.path.exists(filename):
+#            raise ValueError('File {}does not exist, please provide a valid path.'.format(filename))
+#        self.filename=filename
+
+def cread(infile):
+    '''
+    Loads a cifti file and stores as five objects: data array, cifti header, nifti header, extra metadata, filemap
+    Example call:
+        cd,ch,cn,cx,cf=cread(path_to_cifti)
+    
+    Parameters
+    ----------
+    infile: str, Cifti2Image
+        string containing path to cifti file or an already loaded cifti object
+    '''
+    if isinstance(infile,nb.cifti2.cifti2.Cifti2Image):
+        try:
+            cin=infile
+        except:
+            raise ValueError('Please provide a valid cifti file.')
+    elif isinstance(infile,str):
```

Comment:
@elyb01 I think it would be good to also allow `pathlib.Path` objects, in addition to strings, here.